flow/core/rewards.py

```diff
@@ -442,8 +442,7 @@ def miles_per_gallon(env, veh_ids=None, gain=.001):
 
     return mpg * gain
 
-
-def instantaneous_power(env, veh_ids=None, gain=.001):
+def instantaneous_power(env, veh_ids=None, parameter=None, gain=.001):
```

Comment:
I added parameter as an argument of this function in the rewards.py, I will apply the *args / **kwargs in replacement for this parameter.  But I am uncertain, how to pass this parameter value to the initialization of the energy models since I will only be needing this value when I am going to set initial_soc.

Reply:
see above note, so i believe you won't need parameter or gain here anymore.